Tasks/AzureFileCopy/Utility.ps1

```diff
@@ -1275,6 +1275,31 @@ function Is-WinRMCustomScriptExtensionExists
     $isExtensionExists
 }
 
+function Get-TargetUriFromFwdLink { 
```

Comment:
Please merge once you have confirmed that the CDP in master with these changes are green.